### DISCUSSION TITLE
Not sure if bug, or if I'm impacted by the latest release

### DISCUSSION THREAD

**Discussion by andres-lowrie** [neovim/neovim#29214] [Q&A] [answered]
Howdy 👋 

So I just updated to `0.10.0` and I came across the following error when firing up, I've narrowed the issue the following 

```vimscript
vim.go.background = dark
```

when I run it

`nvim -u /Users/andreslowrie/.config/nvim/minmin.init.lua`

```
Error detected while processing /Users/andreslowrie/.config/nvim/minmin.init.lua:                                                                       
E5113: Error while calling lua chunk: /Users/andreslowrie/.config/nvim/minmin.init.lua:1: Cannot unset global option value                              
stack traceback:                                                                                                                                        
        [C]: in function '__newindex'                                                                                                                   
        /Users/andreslowrie/.config/nvim/minmin.init.lua:1: in main chunk   
```

From what I gathered from the man page, `background` is a global option

----

I noticed the release notes mentioning `:colorscheme` and the like so I'm wondering if this is expected or an actual bug

thanks y'all

**Comment by zeertzjq** [comment, marked as answer]
The variable name `dark` does not exist. It should be `vim.go.background = 'dark'`.

**Reply by andres-lowrie** [reply]
lol... how did I miss that? 😅 
thank you
